content/en/docs/getting-started/_index.md

```diff
@@ -70,9 +70,12 @@ sudo systemctl enable crio --now
 To have `systemd` start and manage MicroShift on Podman, run:
 
 ```Bash
-sudo dnf install -y podman
+sudo dnf install -y podman firewalld
 sudo curl -o /etc/systemd/system/microshift.service \
      https://raw.githubusercontent.com/redhat-et/microshift/main/packaging/systemd/microshift-containerized.service
+sudo systemctl enable firewalld --now
+sudo firewall-cmd --permanent --zone=trusted --add-source=10.42.0.0/16
```

Comment:
if we need to enable firewalld we need to add the other ports:

  sudo firewall-cmd --zone=public --add-port=80/tcp --permanent
  sudo firewall-cmd --zone=public --add-port=443/tcp --permanent
  sudo firewall-cmd --zone=public --add-port=5353/udp --permanent
  sudo firewall-cmd --zone=trusted --add-source=10.42.0.0/16 --add-port=53/udp --permanent 
   # For NodePorts to work
  sudo firewall-cmd --zone=public --permanent --add-port=30000-32767/tcp

Reply:
5353 is for MDNS to work

Reply:
If we need this many ports, maybe we should distribute a firewalld service description (e.g. https://github.com/firewalld/firewalld/blob/c11510e065fdbaeb8ff20f32e2628bdd70d30dd7/config/services/bacula.xml), so the instructions become:

- Copy microshift.xml to `/etc/firewalld/services/microshift.xml`
- Enable the service with `sudo firewall-cmd --zone=public --add-service=microshift --permanent`

Reply:
That makes sense, yes.

In general, though, let's please keep the default as locked-down as possible, i.e. the minimum for MicroShift to run, telling users how they should further open up depending on their use case.

For example, I'd argue `sudo firewall-cmd --zone=public --add-port=6443/tcp --permanent` shouldn't be there by default, but opt-in *if* users want `kubectl` access to the kube API from outside the cluster (secure deployments won't allow that) and even then we may be more specific than "anything targeting a port 6443/tcp". Same for https routes of services on the cluster.

Reply:
May be we can ship a couple of xmls, one installed by default (opening the minimal things), another somewhere on the system which can be copied, opening all the service ports, etc to the outside world?

Reply:
sgtm, with clear documentation